### ISSUE TITLE
Xamarin Simple Logger Not Implemented for Windows Phone 8.1 Application

### ISSUE THREAD

**Issue by BestWaySoftware** [jirimatejka/xamarin-logger#3]
Xamarin Simple Logger Not Implemented for Windows Phone 8.1 Application

An exception of type 'System.NotImplementedException' occurred running in a Windows Phone 8.1 Application.

**Comment by jirimatejka** [comment]
That's expected behaviour. Apart from UWP the Windows platform is not supported yet.

**Comment by BestWaySoftware** [comment]
Is there a road map or time frame when it will be implemented?

**Comment by jirimatejka** [comment]
No, it's not. It's not difficult to do, but have a lot of work on another projects.
